lms/static/coffee/src/instructor_dashboard/student_admin.coffee

```diff
@@ -171,6 +180,27 @@ class StudentAdmin
           create_task_list_table @$table_task_history_single, data.tasks
         error: std_ajax_err => @$request_response_error_grade.text full_error_message
 
+    # reset entrance exam attempts for student
+    @$btn_reset_entrance_exam_attempts.click =>
+      unique_student_identifier = @$field_entrance_exam_student_select_grade.val()
+      if not unique_student_identifier
+        return @$request_response_error_ee.text gettext("Please enter a student email address or username.")
+      send_data =
+        unique_student_identifier: unique_student_identifier
+        problem_to_reset: 'entrance_exam'
+        delete_module: false
+      success_message = gettext("Success! Entrance exam attempts reset for student '<%= student_id %>'.")
```

Comment:
Yeah message will be displayed as soon as task is submitted. Will change the message.